server/src/controller/user.ts

```diff
@@ -44,46 +98,42 @@ export class UserController {
         } else {
             // return a BAD REQUEST status code and error message
             ctx.status = 400;
-            ctx.body = "The user you are trying to retrieve doesn't exist in the db";
+            ctx.body = { errors: {error: "The user you are trying to retrieve doesn't exist in the db"}}
         }
-
     }
 
     /**
-     * POST /users
-     * 
-     * Create a user.
+     * GET /api/user/licks
+     *
+     * Get users a lick is shared with by id.
      */
-    public static async createUser(ctx: Context): Promise<void> {
+    public static async getAuthUserLicks(ctx: Context): Promise<void> {
 
-        // get a user repository to perform operations with user
         const userRepository: Repository<User> = getManager().getRepository(User);
+        // will always return the currently authenticated user
+        const authUser: User = await userRepository.findOne({ where: {id: (ctx.state.user.id)}, relations: ['licks']});
+        
+        ctx.status = 200; // OK
+        ctx.body = authUser.licks;
+    }
 
-        // build up entity user to be saved
-        const userToBeSaved: User = new User();
-        userToBeSaved.name = ctx.request.body.name;
-        userToBeSaved.email = ctx.request.body.email;
-
-        // validate user entity
-        const errors: ValidationError[] = await validate(userToBeSaved); // errors is an array of validation errors
+    /**
+     * GET /api/user/licksSharedWithMe
```

Comment:
I think standard convention for URIs is hyphenation.

/api/user/licks-shared-with-me

Not really important though, you can leave this if don't feel like changing it.

Reply:
ok, changed it